### PR TITLE
Allow setting a background color for a gplot (included in PR #186)

### LINKED ISSUE #172: Feature request: add background color
The default plot has a transparent background. If node labels have black font and an IDE with dark mode is being used, it's hard to read the plot. It would be nice to be able to set the background to white, for instance.

### PR DESCRIPTION
**Pull request by hdavid16**
Fixes #172

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,6 +183,7 @@ gplot(h)
 + `arrowangleoffset` Angular width in radians for the arrows. Default: `π/9 (20 degrees)`
 + `linetype` Type of line used for edges ("straight", "curve"). Default: "straight"
 + `outangle` Angular width in radians for the edges (only used if `linetype = "curve`). Default: `π/5 (36 degrees)`
++ `backgroundc` Color for the plot background. Default: `nothing`
```
Comment (simonschoelly) [inline README.md:186]: I know that for other params we alse use the `c` postfix, but maybe we should just call it `background_color`.

**Comment by simonschoelly** [comment]
Would it be possible to add a test for the background color option?

File: README.md (hunk 1 above)
Comment (hdavid16) [reply README.md:186]: yes! Should we think about changing the kwargs to being more verbose? `nodefill_color`, etc.

**Comment by hdavid16** [comment]
> Would it be possible to add a test for the background color option?

I just added a fourth test in the karate test, where the background color is set to lightyellow

**Pull request closed without merging**